README.md

```diff
@@ -36,27 +37,61 @@ In particular, you can provide them with `--release`, `--dev`, or `--profile`
 flags to switch the compilation profile. If not option is provided, the scripts
 default to the `--release` profile.
 
-### Running application and examples
 For best experience, it is recommended to use the 
-`scripts/watch.sh --dev` in a second shell. In order to build the IDE application, 
-follow the steps below:
+`scripts/watch.sh --dev` in a second shell.
+
+### Web Application
```

Comment:
As we get this more complex than before, we should make this doc chronological

1. Building Rust Sources.
2. Minimizing the WASM binary size (optional).
3. Running / Packaging the App.

Lets use these 3 titles to group the steps, ok? :)

:

Reply:
@notdanilo ?